core/common/src/tile/TileMetadata.ts

```diff
@@ -771,7 +771,7 @@ export function readTileContentDescription(stream: ByteStream, sizeMultiplier: n
     // Must sub-divide if tile explicitly specifies...
     let canSkipSubdivision = 0 === (header.flags & ImdlFlags.DisallowMagnification);
     // ...or in 2d, or if app explicitly disabled magnification, or tolerance large enough to risk quantization error...
```

Comment:
does this comment need to be updated to indicate the removal of the explicitly-disabled-magnification condition
or moved to the context below where the magnification is consulted?

Reply:
reverted - that change doesn't belong in this PR.